README.md

```diff
@@ -234,100 +240,350 @@ The complete list of transformations supported and their usage in ImageKit can b
 | effectGray | e-grayscale |
 | original | orig |
 
-
-
 ### File Upload
 
 The SDK provides a simple interface using the `.upload()` method to upload files to the ImageKit Media Library. It accepts all the parameters supported by the [ImageKit Upload API](https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload).
 
 The `upload()` method requires at least the `file` and the `fileName` parameter to upload a file. You can pass other parameters supported by the ImageKit upload API using the same parameter name as specified in the upload API documentation. For example, to specify tags for a file at the time of upload, use the `tags` parameter as specified in the [documentation here](https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload).
 
 Sample usage
+
 ```cs
-ImagekitResponse resp = await imagekit
-    .FileName("my_file_name.jpg")
-    .UploadAsync(<fullPath|url|base_64|binary>);
+ResponseMetaData resp = await imagekit.UploadAsync(<fullPath|url|base_64|binary>,"my_file_name.jpg");
 ```
 
-**Note**: Upload argument can be a local fullPath or URL or byte array (byte[]) or Base64String of a file.
```

Comment:
Change wording of `List of supported transformations` to let users know that these are methods. Also, update the table with correct method names. Do include an example of `Raw`.